docs/conceptual/array.sum[^t]-function-[fsharp].md

```diff
@@ -36,15 +36,16 @@ The input array.
 
 
 ## Return Value
-
-The resulting sum.
+The sum of all elements of the array.
 
 
 ## Exceptions
-May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception.aspx), as the function uses the [checked (+) operator](https://msdn.microsoft.com/visualfsharpdocs/conceptual/checked.[-p-][%5Et1%2c%5Et2%2c%5Et3]-function-[fsharp]) for the additions.
+May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception.aspx) due to arithmetic overflows.
```

Comment:
I would still keep the previous version here, with the `checked` operator reference. It's good to be concise, but these are developer docs, so we need to make sure we cover why things are happening.